### DISCUSSION TITLE
Issue while pushing to github

### DISCUSSION THREAD

**Discussion by 22f2000893** [community/community#123479] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

# The error I'm getting when pushing to github:
`kex_exchange_identification: banner line contains invalid characters\r\nbanner exchange: Connection to 64:ff9b::14cf:4952 port 22: invalid format\r\nfatal: Could not read from remote repository.\n\nPlease make sure you have the correct access rights\nand the repository exists`
![Screenshot 2024-05-12 085326](https://github.com/community/community/assets/140406492/8e3c79c4-8736-4d9c-aac3-62253fc76b9b)
![Screenshot (58)](https://github.com/community/community/assets/140406492/796294db-791c-401f-8f33-cdafefce442b)

**Comment by CiceroLino** [comment, marked as answer]
First of all, have you sync your github account to vscode? Normally, github access is stored when you sync yout github account to vscode.

If dont, you must have create a access token to use like password

**Reply by 22f2000893** [reply]
Thank you, I have two factor authorization on and also haven't set personal access token.
